packages/contracts-bedrock/contracts/deployment/MigrationSystemDictator.sol

```diff
@@ -183,12 +188,28 @@ contract MigrationSystemDictator is BaseSystemDictator {
     /**
      * @notice Tranfers admin ownership to the final owner.
      */
-    function step6() external onlyOwner step(6) {
+    function finalize() external onlyOwner {
         // Transfer ownership of the L1CrossDomainMessenger to the final owner.
         L1CrossDomainMessenger(config.proxyAddressConfig.l1CrossDomainMessengerProxy)
             .transferOwnership(config.globalConfig.finalOwner);
 
         // Transfer ownership of the ProxyAdmin to the final owner.
         config.globalConfig.proxyAdmin.transferOwnership(config.globalConfig.finalOwner);
+
+        // Optionally also transfer AddressManager and L1StandardBridge if we still own it. Might
+        // happen if we're exiting early.
+        if (currentStep <= 3) {
```

Comment:
this feels a little bit brittle to future changes in the steps. What about introducing a new `uint constant BREAKPOINT1 = 3` to make it a bit more explicit?